server/index.js

```diff
@@ -1,5 +1,6 @@
+const { appInsights } = require('./utils/azureAppInsights')
```

Comment:
Moved this to the top, I know they usually say to import it as early as possible 

https://www.npmjs.com/package/applicationinsights
`Important: applicationinsights must be setup and started before you import anything else. There may be resulting telemetry loss if other libraries are imported first.`